packages/manager/apps/telecom/src/app/telecom/pack/move/building-details/move-building-details.html

```diff
@@ -0,0 +1,163 @@
+<div>
```

Comment:
Useless
```suggestion
```
Also, there are many useless `<div>` in this file. (Maybe rests of previous `div` elements with removed classes)